server/.dockerignore

```diff
@@ -11,4 +11,5 @@
 !tsconfig.build.json
 !prisma
 !scripts
-!.env*
\ No newline at end of file
+!.env*
+!docker-entrypoint.sh
```

Comment:
Can we invert this ? Include everything but the commonly excluded files (node_modules, transpiled files, etc) Less likely to change.

Reply:
I prefer using the .dockerignore as a whitelist to be explicit about what's in the image. Yes it's more likely to change, but it's easy to spot that a file is missing in the docker context when trying to build the image.